dist/bin/current/roddy.sh

```diff
@@ -129,7 +129,23 @@ elif [[ "$parm1" == "createworkflow" ]]; then
 fi
 
 IFS=""
-#[[ $RMIPORT != "" ]] && export DBG_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005" && echo "Opened up rmi debugger port"
-java $DBG_OPTS -cp .:$libraries:${RODDY_BINARY} de.dkfz.roddy.Roddy $*
-# -verbose:class
+export RGROOVY_LIB=`readlink -f ${RODDY_BINARY_DIR}/lib/groovy*.jar`
+
+# Check for groovyserv. If it does not exist, try to download it. If it still fails, start Roddy without it.
+caller=java
```

Comment:
Can you write an error/warning message that describes what to do (where to put the gservforbidden-file with what content), if a timeout occurs?